### PR TITLE
DM-42810: Extend Qserv integration test to support the HTTP-based front-end

### PR DESCRIPTION
**Pull request by iagaponenko**

### CODE REVIEW

File: admin/local/docker/compose/docker-compose.yml [hunk 1]
```diff
@@ -41,30 +41,37 @@ x-worker-repl:
...
-    czar_0_data:
-    czar_0_cfg:
-    czar_0_home:
-    czar_0_mariadb_lib:
-    czar_0_mariadb_log:
-    czar_0_mariadb_run:
-    repl_mgr_data:
-    repl_mgr_home:
-    repl_mgr_mariadb_lib:
+    volume_czar_xrootd:
```
Comment (fritzm) [inline admin/local/docker/compose/docker-compose.yml:44]: Is `volume_` prefix here redundant, given that inspection or specification contexts where these would be used are all exclusive to volumes?

If you think having it there makes things more understandable generally, then I am fine with it.  Understandability is "king".  Just asking!

File: src/admin/templates/http/etc/qserv-czar.cnf.jinja [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+user = qsmaster
+port = {{ czar_db_port }}
+maxMsgSourceStore = 5
+
+# database connection for QStatusData
+# Using qmeta for now. May move this qservResult or make its own instance
+# databse based on what happens with multiple czars.
+[qstatus]
+passwd =
+#db = qservStatusData
```
Comment (fritzm) [inline src/admin/templates/http/etc/qserv-czar.cnf.jinja:51]: Remove dead wood comment?

File: src/admin/python/lsst/qserv/admin/qservCli/launch.py [hunk 3]
```diff
@@ -32,11 +32,11 @@
 from typing import Dict, List, Optional
 
 from .opt import (
-    dashboard_port_ev,
-    mariadb_image_ev,
-    qserv_image_ev,
-    dh_user_ev,
-    dh_token_ev,
+    envval_dashboard_port,
```
Comment (fritzm) [inline src/admin/python/lsst/qserv/admin/qservCli/launch.py:35]: Slight preference for the slightly shorter `env_` prefix here throughout, but leave to your discretion :-)

Comment (iagaponenko) [reply src/admin/python/lsst/qserv/admin/qservCli/launch.py:35]: Good point! Will fix that.

File: src/admin/templates/http/etc/qserv-czar.cnf.jinja (hunk 2 above)
Comment (iagaponenko) [reply src/admin/templates/http/etc/qserv-czar.cnf.jinja:51]: Will do it.

File: admin/local/docker/compose/docker-compose.yml (hunk 1 above)
Comment (iagaponenko) [reply admin/local/docker/compose/docker-compose.yml:44]: Let me explain my motivation by using `czar_xrootd` as an example. If you browse the original (before my modifications) code for all occurrences of the name you would find 14 of those in 4 different contexts.

The first context is the name of a volume defined as:
```
volumes:
    czar_xrootd:
```
The identifier is referenced at (and other locations):
```
    czar_cmsd:
        ...
        volumes:
            - type: volume
              source: volume_czar_xrootd
```

The second context is the base name of a container:
```
    czar_xrootd:
        image: "${QSERV_IMAGE:?err}"
        init: true
        command: >
        ...
```
The third context is the virtual hostname (which is not the same as the name of a container):
```
    czar_xrootd:
        image: "${QSERV_IMAGE:?err}"
        ...
        hostname: czar_xrootd
```
The fourth context is the network mode (which could be related to the hostname?):
```
    czar_cmsd:
        image: "${QSERV_IMAGE:?err}"
        ...
        network_mode: "service:czar_xrootd"
```
So, the name is clearly overloaded. In my naive understanding, such overloading is probably normal for YAML, where the same name is treated differently depending on a context (where it's defined, and where it's referenced). However, I find it confusing and counterintuitive. Hence was my attempt to modify the configuration to prefix the class name (where class = "volume") in the name of an object.

BTW I run into a similar (name overloading) problem in the Python code.

Perhaps, for the "seasoned" YAML developer the overloading of names is not a problem.

Comment (fritzm) [reply admin/local/docker/compose/docker-compose.yml:44]: I see -- seems reasonable!

**Review by fritzm — APPROVED** [review]
LGTM -- thanks!

**Pull request merged**